resource-managers/kubernetes/core/src/test/scala/org/apache/spark/scheduler/cluster/kubernetes/KubernetesTaskSetManagerSuite.scala

```diff
@@ -26,6 +26,7 @@ import org.apache.spark.scheduler.{FakeTask, FakeTaskScheduler, HostTaskLocation
 
 class KubernetesTaskSetManagerSuite extends SparkFunSuite {
 
+  SparkContext.clearActiveContext()
```

Comment:
Again - should this be being done in a `before` block? Also, can we use a mock `SparkContext`?